apps/src/templates/projects/FeaturedProjects.jsx

```
@@ -8,23 +8,30 @@ import {
 
 export default class FeaturedProjects extends React.Component {
   static propTypes = {
-    currentFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
+    activeFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
       .isRequired,
-    archivedUnfeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
+    archivedFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
+      .isRequired,
+    savedFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
       .isRequired,
   };
 
   render() {
     return (
       <div>
-        <h3>Currently Featured Projects</h3>
+        <h3>Active Featured Projects (currently displayed in Gallery)</h3>
         <FeaturedProjectsTable
-          projectList={this.props.currentFeaturedProjects}
+          projectList={this.props.activeFeaturedProjects}
           tableVersion={featuredProjectTableTypes.current}
         />
+        <h3>Saved Featured Projects</h3>
+        <FeaturedProjectsTable
+          projectList={this.props.savedFeaturedProjects}
+          tableVersion={featuredProjectTableTypes.saved}
+        />
```

The PR description notes that the number of featured projects makes the tables unwieldy to scroll. I also think this set up might get confusing in the long term.

 I'm wondering if it would be easier to start the curation of projects from a clean slate? Maybe something like: 
- delete everything in the `FeaturedProjects` table 
- rename FeaturedProjects to `ShortlistedProjects` (or `SavedProjects` or `NotableProjects` or whatever...)
- then display 1 table of `ShortlistedProjects` where each project has a status of "featured", "archived" or "never featured" 

This is a big deviation from what you have, though, so let me know if you want to talk it over or skip altogether! Mostly thinking about maintainability and how to avoid needing a lot of historical knowledge to unpack the meaning of these tables.

---

@samantha-code - lmk what you think - thanks!

---

I dont want to remove the projects in the current featured projects table as the basis for shortlisted/saved projects so I have a backlog to feature in the future. I agree that there are a ton of tables and it is a big tedious. I love the idea of a single table, but I would like the ability to filter the table on the "status" so that I could see all featured projects together etc. (which is what a single table gets us).

---

In the follow-up section, I propose that we add a view/hide button for each of the three tables that would expand/collapse the tables - the default could be to hide the list. That's another option.

---

Update: after chatting with @samantha-code , I'm going to refactor...
- Use a table named 'favorite_projects' instead of 'featured_projects'
- Three statuses: 'featured', 'never_featured', and 'previously_featured'
- Display in one table at 'projects/featured'
- Quick actions will be different depending on status (may hide/disable options that are not offered for a particular status)
- Display columns: 'Project Name', 'Type', 'Topic', 'Featured', 'Unfeatured', 'Quick Actions' ('Featured' and/or 'Unfeatured' could be blank depending on status of project)